tests/test_cmd_resetdep.py

```diff
@@ -0,0 +1,76 @@
+import pytest
+from six import StringIO
+
+from doit.cmd_resetdep import ResetDep
+from doit.dependency import TimestampChecker
+from doit.exceptions import InvalidCommand
+from doit.task import Task
+from tests.conftest import tasks_sample, CmdFactory
+
+
+class TestCmdResetDep(object):
+
+    def test_execute(self, pdepfile, dependency1):
+        output = StringIO()
+        tasks = tasks_sample()
+        cmd_list = CmdFactory(ResetDep, outstream=output, task_list=tasks,
+                              dep_manager=pdepfile)
+        cmd_list._execute()
+        got = [line.strip() for line in output.getvalue().split('\n') if line]
+        expected = ["processed %s" % t.name for t in tasks]
+        assert sorted(expected) == sorted(got)
+
+    def test_file_dep(self, pdepfile, dependency1):
+        my_task = Task("t2", [""], file_dep=['tests/data/dependency1'])
+        output = StringIO()
+        cmd_list = CmdFactory(ResetDep, outstream=output, task_list=[my_task],
+                              dep_manager=pdepfile)
+        cmd_list._execute()
+        got = output.getvalue()
+        assert "processed t2\n" == got
```

Comment:
Apart from checking the output "saying" the new value was saved, can it also check that the new value was actually saved ?